datafusion/core/src/scheduler/task.rs

```diff
@@ -35,7 +35,7 @@ use std::sync::{Arc, Weak};
 use std::task::{Context, Poll};
 
 /// Spawns a [`PipelinePlan`] using the provided [`Spawner`]
-pub fn spawn_plan(plan: PipelinePlan, spawner: Spawner) -> ExecutionResults {
+pub(crate) fn spawn_plan(plan: PipelinePlan, spawner: Spawner) -> ExecutionResults {
```

Comment:
This is a private module and so this change is practically unnecessary, but needed to placate clippy